### PR TITLE
Updates to MySQL Agent, to work with PHP7 (mysql -> mysqli)

### PR DESCRIPTION
**Pull request by arrmo**
To this issue,
https://github.com/librenms/librenms/issues/3751#issuecomment-231546464

Yell if you seen anything wrong. Thanks!

### CODE REVIEW

File: agent-local/mysql [hunk 1]
```diff
@@ -270,18 +270,18 @@ function ss_get_mysql_stats( $options ) {
...
-      $conn = mysql_connect($host_str, $user, $pass, true, MYSQL_CLIENT_SSL);
+      $conn = mysqli_connect($host_str, $user, $pass, true, MYSQL_CLIENT_SSL);
    }
    else {
-      $conn = mysql_connect($host_str, $user, $pass);
+      $conn = mysqli_connect($host_str, $user, $pass);
    }
    if (!$conn ) {
-      die("MySQL: " . mysql_error());
+      die("MySQL: " . $conn->connect_error);
```
Comment (paulgear) [inline agent-local/mysql]: How are we going to get $conn->connect_error if $conn is null?

Comment (arrmo) [inline agent-local/mysql]: Good catch! My mistake. Checking out this, http://php.net/manual/en/mysqli.error.php, and the code there,
if ($mysqli->connect_errno) {
    printf("Connect failed: %s\n", $mysqli->connect_error);
    exit();
}

Seems the code above should be as follows, agreed?
if ($conn->connect_errno) {

I admit - I had it running, but missed checking with MySQL down. Again, thanks!

Comment (laf) [inline agent-local/mysql]: yeah that looks good.

**Comment by laf** [comment]
@arrmo thanks for doing this, @murrant also submitted a fix and as that didn't need any further changes we've merged that one.

**Comment by arrmo** [comment]
NP - works for me. Thanks!

**Pull request closed without merging**